src/atlas/functionspace/PointCloud.h

```diff
@@ -41,11 +41,9 @@ class PointCloud : public functionspace::FunctionSpaceImpl {
 public:
     template <typename Point>
     PointCloud(const std::vector<Point>&);
-    PointCloud(const Field& lonlat);
-    PointCloud(const Field& lonlat, const Field& ghost);
-
-    PointCloud(const FieldSet&);  // assuming lonlat ghost ridx and partition present.
-
+    PointCloud(const Field&);
+    PointCloud(const Field&, const Field&);
```

Comment:
It is good to keep the arguments named in above 2 lines to indicate what is expected without going into the ".cc" file.

Reply:
... done